### PR TITLE
Auto enable TTS autoplay

### PR DESCRIPTION
**Pull request by maureensturgeon**
When editing a section, when the course is selected, if it is a pre-reader course AND the teacher is on Chrome, IE, or Edge then turn TTS autoplay on by default. 

## Testing story
- Added unit tests
- Manually tested that adding a pre-reader course on Safari does not turn on autoplay
- Manually tested that adding a pre-reader course on Chrome does turn on autoplay
- Manually tested on Chrome that adding a non pre-reader course does not turn on autoplay

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by jamescodeorg — APPROVED** [review]
Change looks good to me, but did we want to let the feature "bake" a bit before turning it on by default?

**Comment by maureensturgeon** [comment]
> Change looks good to me, but did we want to let the feature "bake" a bit before turning it on by default?

Yeah I wanted to have this ready, but I think I may wait until next week to merge it. I'll make sure the branch stays up-to-date.

**Comment by maureensturgeon** [comment]
We have not determined when/if this feature should be implemented. Closing for now.

**Pull request closed without merging**
